lib/parallel_tests/cucumber/runner.rb

```diff
@@ -28,7 +28,11 @@ def summarize_results(results)
 
           output << super
 
-          output.join("\n\n")
+          consolidated_results[:summary] = {
```

Comment:
output is everything the command prints, not sure why it needs to be in the json ... will make kit so much bigger ...